src/cf-logstash-filters/src/logstash-filters/snippets/app-metric.conf

```diff
@@ -1,7 +1,7 @@
 ##---------------------------------------------------------------------
 # Parses message with app metric data.                                 | |
 ##---------------------------------------------------------------------
-if ( [metrics_sd_id] == "gauge@47450" ) {
+if ( "" in [metrics_sd_id] ) {
```

Comment:
Can we add a comment explaining what this does? For some reason my brain struggles to parse it